build_tools/ci/run_matmul_test.sh

```diff
@@ -206,19 +206,22 @@ function run_matmul_test() {
   # The default is to not expect a compilation failure.
   local expect_compile_failure="0"
 
-  # The default is to compile and run the numerical test.
-  local compile_only="0"
```

Comment:
I think it's good to keep this flag

Reply:
Ok, but the reason this flag is removed in this PR is that it is equivalent to n_repeats=0 (and so redundant)

Reply:
Yeah, I figured that, but it seems to be more readable to use flag `compile_only`. Maybe to change `n_repeats` to `num_repeat_runs`?